iff --git a/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js b/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
index 46634a1..1102076 100644
--- a/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
+++ b/node_modules/@rabby-wallet/eth-trezor-keyring/dist/index.js
@@ -108,7 +108,7 @@ class TrezorKeyring extends events_1.EventEmitter {
                 this.model = event.payload.features.model;
             }
         });
-        trezor_connect_1.default.init({ manifest: TREZOR_CONNECT_MANIFEST });
+        trezor_connect_1.default.init({ manifest: TREZOR_CONNECT_MANIFEST, webusb: false });
     }
     /**
      * Gets the model, if known.
